tensorflow/core/kernels/BUILD

```diff
@@ -995,7 +995,9 @@ tf_kernel_library(
         ":bounds_check",
         "//tensorflow/core:framework_lite",
         "//third_party/eigen3",
-    ],
+    ] + if_sycl([
```

Comment:
We cannot do this, as `:sycl_runtime` itself depends on the full `:framework`, so this would force `:framework_lite` to depend on `:framework` itself.